### ISSUE TITLE
PieChart : Donut : Label greater than 20 chars are getting trimmed and itemclick is failing

### ISSUE THREAD

**Issue by maildp** [swimlane/ngx-charts#1094]
**I'm submitting a ...**  (check one with "x")

- [X ] bug report - search github for a similar issue or PR before submitting
- [ ] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**
In advanced pie chart i.e. donut chart , if the length of character is greater than 20, its is displayed with 3 dots at the end.

Issue 1 : Since tooltip support is not there, user cannot read that label completely.
Issue 2 ; The Itemclick in the legend label provides incorrect value. It sends the name value as displayed in the label. So if chars in the label is more than 20 , it is displayed as 20 chars and 3 dots and same value is sent in the itemclick event.

Ex let the name value be "test for 20 characters"
Value that is displayed : "test for 20 characte..."
Value for name in itemclick event : "test for 20 characte..."

Check the screenshot and focus on highlighted area : 

![donut](https://user-images.githubusercontent.com/39160275/52035429-83c75180-2550-11e9-8842-f278361de91f.png)

**Expected behavior**

For issue1 : tooltips should be supported
For issue 2 : itemclick should provide correct valu

**Reproduction of the problem**
Just provide data with more than 20 chars

**Please tell us about your environment:**
Windows

* **ngx-charts version:** x.x.x
Latest version

* **Angular version:** x.x.x
Version 6

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
I checked on chrome and explorer, but i think it can be reproduced in every browser

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by maildp** [comment]
Github link to download project : https://github.com/swimlane/ngx-charts

**Comment by VladBozhinovski** [comment]
Any update on this issue ?

**Comment by KingDarBoja** [comment]
@VladBozhinovski The referenced PR does solve one of the issues (as stated by the author) but that's not the one that you wanted, right?

Cheers!

**Comment by VladBozhinovski** [comment]
> @VladBozhinovski The referenced PR does solve one of the issues (as stated by the author) but that's not the one that you wanted, right?
> 
> Cheers!

Not really, I just need it without trimming the labels. The posted fix is about truncated label which is different issue as I can see. Anyway, thanks for responding.

Cheers!
